package.json

```diff
@@ -76,9 +78,9 @@
 		"test:all": "yarn turbo run test --filter=!render --filter=!rrweb --filter=!@rrweb/types --filter=!rrweb-snapshot --filter=!rrweb-player --filter=!rrdom --filter=!rrdom-nodejs --filter=!nextjs",
 		"test:render": "yarn turbo run test --filter=render",
 		"sourcemaps:frontend": "yarn workspace @highlight-run/frontend sourcemaps",
-		"knip:frontend": "knip --workspace frontend"
+		"knip:frontend": "knip --workspace @highlight-run/frontend"
 	},
-	"devDependencies": {
+	"dependencies": {
```

Comment:
Just curious, what's the reason for this change?

Reply:
was trying to figure out why `turbo` added as a top level `dev` dependency was not accessible to sub projects (ie. rrweb, having to add `turbo` to each subpackage). this didn't help tho so i'll undo it